### ISSUE TITLE
Fix accessibility issue: no aria text for namespace selection

### ISSUE THREAD

**Issue by a-roberts** [tektoncd/dashboard#524]
# Expected Behavior
An accessibility assessment tool (I've used the IBM DAP tool) should report no accessibility problems

# Actual Behavior
There's one
![image](https://user-images.githubusercontent.com/12814972/64952478-5d082200-d878-11e9-9823-2deaf3ce1cff.png)
this is for the namespace dropdown on the left

# Steps to Reproduce the Problem

1. Run the IBM DAP tool (note this may be IBMer specific, it's what myself and @carlos-logro have been using to good effect, open to better suggestions that are more widely available)
2. Observe violation reported when scanning a deployed Tekton Dashboard with Webhooks Extension

For clues have a look at https://github.com/tektoncd/dashboard/blob/bcfe703b8af6fd74993d897efb5bf0e93ad6bed6/src/containers/NamespacesDropdown/NamespacesDropdown.js - should just set the accessible text to be what the current element is I think

http://react.carbondesignsystem.com/?path=/story/dropdown--default see here also. If I make the ARIA label say "Namespace *namespace name* and rescan, it goes away, I'll fix, figuring out how to do this through code now instead of just using the inspector.

**Comment by a-roberts** [comment]
This is even the case with http://react.carbondesignsystem.com/?path=/story/dropdown--default, issue raised (linked to above)...

**Comment by a-roberts** [comment]
Putting this back on the backlog, heard nothing back for the issue I raised against Carbon and focused on bug fixing atm

**Comment by a-roberts** [comment]
Oddly enough this is now fixed (verified with a rescan using the DAP tool and using Voiceover - the namespace dropdown and its elements are now read out correctly), closing
